src/main.c

```diff
@@ -162,11 +141,17 @@ void handleGetWalletId(volatile unsigned int *tx) {
     unsigned char t[64];
```

Comment:
IMO HAVE_WALLET_ID_SDK should be dropped. See https://ledgerhq.atlassian.net/browse/EMBSDK-18

Reply:
:thinking: Should be part of a dedicated ticket and PR, I guess?